libs/domain/site/src/models/breadcrumbs.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Breadcrumb {
+  text?: string;
```

Comment:
What about having generic text resource, that can either be a string or object with token?

Reply:
dunno, lets ask Tobi when he is back